docs/GLOSSARY.md

```diff
@@ -0,0 +1,12 @@
+## complete native application descriptor
+
+The complete native application descriptor is a string with format `[nativeAppName]:[platform]:[version]` 
+- `nativeAppName` : alphanumeric native application name , cannot contain character ':'
+- `platform` : one of `android` or `ios`
+- `version` : alphanumeric , cannot contain character ':'
+
+## partial native application descriptor
+
+The complete native application descriptor is a string with format `[nativeAppName]:[platform]` 
```

Comment:
Same remark as previous, but also fix `complete` (its `partial` section here), bad copy paste ;)

Instead of `The complete`, should be `A partial`

Reply:
Also partial can be only the native app name, i.e `[nativeAppName]`